lib/take_elevated.lua

```diff
@@ -133,4 +163,4 @@ if drip_interval > 0 then
 end
 
 -- Return the current quota
-return { bucket_content_after_take, enough_tokens, current_timestamp_ms, reset_ms, erl_triggered, is_erl_activated, erl_quota_left }
+return { bucket_content_after_take, enough_tokens, redis_timestamp_ms, reset_ms, erl_triggered, is_erl_activated, erl_quota_left }
```

Comment:
Can you configure your IDE to always add a newline at the end of the file?

Simpler diffs, according to the POSIX standard a line in a text file should always end up with a newline (and a side effect of this is that many tools won't process the last line properly if it is not ending with a newline, like `wc` (will not count the last line) or `read` (won't read the last line, see [this PR](https://github.com/atko-cic/belgrano/pull/260)).

Reply:
fixed